src/Domain/Entities/CompanyHierarchy/Location.cs

```diff
@@ -13,6 +13,7 @@ public class Location : ICHNodeWithParent<Station>
     public int ParentId { get; set; }
     public Detector? Detector { get; set; }
     public List<Task> Tasks { get; set; } = default!;
+    public List<CalibrationCoordinates>? OriginalCoordinates { get; set; } = default!;
```

Comment:
Ez valoszinuleg nem fog igy mukodni. Hasonlo szitu mint TaskInstance.Remaining-nel, csak ezt kellemetlenebb lenne stringbe kepezni. 

Relevans doc page: https://learn.microsoft.com/en-us/ef/core/modeling/owned-entities#collections-of-owned-types